### DISCUSSION TITLE
Why is this extension being duplicated into another one?

### DISCUSSION THREAD

**Discussion by illtellyoulater** [imrayya/stable-diffusion-webui-Prompt_Generator#8] [General]
I just found this:

https://github.com/AUTOMATIC1111/stable-diffusion-webui-promptgen

so which one should we use now?

**Comment by imrayya** [comment]
I have no idea of the reasoning why [AUTOMATIC1111 ](https://github.com/AUTOMATIC1111/stable-diffusion-webui-promptgen) made a very similar extension. We haven't communicated it about. His UI seems very clean though and it nice to add the ability to add different arbitrary models to it. 

I might do a pull request into the future to consolidate my 'features' into the other extension. No ETA on that though

And use whichever suits your needs the most. You could download both models that mine currently implements ([FredZhang7/distilgpt2-stable-diffusion-v2](https://huggingface.co/FredZhang7/distilgpt2-stable-diffusion-v2) and [Gustavosta/MagicPrompt-Stable-Diffusion](https://huggingface.co/Gustavosta/MagicPrompt-Stable-Diffusion)) and solely use AUTOMATIC1111's extension. The other way around isn't possible currently.

**Reply by illtellyoulater** [reply]
Thanks, I wrote to him too, just in case he is not aware...
